docs/docs/dev_docs/getting_started/quickstart.md

```diff
@@ -32,6 +32,14 @@ With the help of Aztec.js you will be able to:
 
 This quickstart walks you through installing the Sandbox, deploying your first Noir contract, and verifying its execution!
 
+## Sandbox Contents
+
+The sandbox contains a local ethereum instance running [Anvil](https://book.getfoundry.sh/anvil/), a local instance of the Aztec rollup, an aztec private execution client for handling user transactions and state, and, if using Docker, an (Otterscan)[https://github.com/otterscan/otterscan] block explorer for the local ethereum network.
```

Comment:
Same thing for the link up here, see comment below.

Reply:
d'oh